### ISSUE TITLE
Argument "type" validation

### ISSUE THREAD

**Issue by crookse** [drashland/line#19]
## Summary

What:

Currently, the argument validation only checks to see that the arguments were specified:

 - If an argument is missing, then an error shows up.
 - If an extra argument was passed in, then an error shows up.

We should do argument validation on the "type" of the argument. For example, I want to make 10 test files using `rhum` like this:

```
rhum make 10
```

With the above command, Line should check that `10` is a number.

Why:

It would be nice to have built-in argument type validation.

## Acceptance Criteria

Below is a list of tasks that must be completed before this issue can be closed.

- [ ] Write documentation
- [ ] Write unit tests
- [ ] Write integration tests

### Signature property

```typescript
public signature = "test-creator [test_filename: string] [num_test_cases: number]";
```

### Help Menu

```text
Test Creator - Make test files

USAGE

    test-creator [option]
    test-creator [arg: test_filename] [arg: num_test_cases]

ARGUMENTS

    test_filename <string>
        The test's filename.
    num_test_cases <number>
        The number of test cases to write in the test file.

OPTIONS

    -h, --help
        Show this menu.
    -v, --version
        Show this CLI's version.
```

**Comment by ebebbington** [comment]
would be really cool
